src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -641,6 +691,51 @@ internal static string NormalizeFilePath(string path)
             return Path.GetFullPath(path);
         }
 
+        /// <summary>
+        /// Determine the execution policy based on the supplied string.
+        /// If the string doesn't match to any known execution policy, set it to incorrect.
+        /// </summary>
+        /// <param name="_executionPolicy">The value provided on the command line.</param>
+        /// <returns>The execution policy.</returns>
+        private static ParameterBitmap GetExecutionPolicy(string? _executionPolicy)
+        {
+            if (_executionPolicy is null)
+            {
+                return ParameterBitmap.EPUndefined;
+            }
+
+            ParameterBitmap ExecutionPolicySetting = ParameterBitmap.EPUndefined;
+            switch (_executionPolicy)
+            {
+                case var b when string.Equals(_executionPolicy, "default", StringComparison.OrdinalIgnoreCase):
```

Comment:
```suggestion
                case bool b when string.Equals(_executionPolicy, "default", StringComparison.OrdinalIgnoreCase):
```

Reply:
it's not a bool, it's actually a string - it's a new C# 7 thingy that surprisingly returns a string in this context.

Reply:
What ever type it returned, it would be nice to make it explicit.  But not a blocker.

Reply:
and it's not used - I imagine (hope) that the compiler will just optimize it away since it's not used